proxy/http/HttpTransact.cc

```diff
@@ -3639,11 +3651,13 @@ HttpTransact::handle_response_from_parent(State *s)
       TRANSACT_RETURN(SM_ACTION_DNS_LOOKUP, PPDNSLookup);
       break;
     case ORIGIN_SERVER:
-      s->current.attempts = 0;
+      // Do DNS as we are now going to Origin Server
+      TRANSACT_RETURN(SM_ACTION_DNS_LOOKUP, OSDNSLookup);
+      /*s->current.attempts = 0;
```

Comment:
If this is dead code, either #ifdef it, or just remove it. These comments makes it difficult to see the intent here :).